tools/github/scripts/Snapshot-AzsdkLabels.ps1

```diff
@@ -0,0 +1,150 @@
+[CmdletBinding(DefaultParameterSetName = 'RepositoryFile', SupportsShouldProcess = $true)]
+param (
+    [Parameter()]
+    [ValidateScript({Test-Path $_ -PathType 'Container'})]
+    [string]$SnapshotDirectory = "$PSScriptRoot/../data/repository-snapshots",
+
+    [Parameter()]
+    [ValidateScript({Test-Path $_ -PathType 'Leaf'})]
+    [string] $LabelsFilePath = "$PSScriptRoot/../data/common-labels.csv",
+
+    [Parameter(ParameterSetName = 'RepositoryFile')]
+    [ValidateScript({Test-Path $_ -PathType 'Leaf'})]
+    [string] $RepositoryFilePath = "$PSScriptRoot/../data/repositories.txt",
+
+    [Parameter(ParameterSetName = 'Repositories')]
+    [ValidateNotNullOrEmpty()]
```

Comment:
Why have 3 different way to specify these? Why not just use the repositories.txt?

Reply:
These are just following the pattern that was previously established.  We do occasionally run one-offs for subsets, so having an override is helpful.   I'm not sure how @heaths originally settled on using both the full repository name and the language shorthand.